help_function/src/swap_device_vector.cpp

```diff
@@ -0,0 +1,184 @@
+#include <oneapi/dpl/execution>
```

Comment:
Do we need a copyright notice at the beginning of this file? Some of the other files touched in this PR also lack this notice.

Reply:
Thanks, fixed.